yt/visualization/color_maps.py

```diff
@@ -48,26 +52,19 @@ def register_yt_colormaps_from_cmyt():
     """
 
     for hist_name, alias in _HISTORICAL_ALIASES.items():
-        cmap = mpl.colormaps[alias].copy()
-        cmap.name = hist_name
-        mpl.colormaps.register(cmap)
-        mpl.colormaps.register(cmap.reversed())
+        cmap = mpl.colormaps[alias]
```

Comment:
note for later that this even if we close this PR, this line change is worth using if only because `mpl.colormaps.__getitem__` *already* returns copies, so the current implementation of this line is unjustified.